### ISSUE TITLE
Plugin not working after upgrade to v1.1.2

### ISSUE THREAD

**Issue by Vishnupriyaakinapelli** [react-theming/storybook-addon#34]
We have been using react-theming/storybook-addon for quite some time and it was working fine. After recent upgrade to v1.1.2, the plugin is not working and we are seeing theming [tab](url) with 'Waiting for data' message. Also, below console error is seen when we click on the theming tab.

vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'setQueryParams')
    at setCurrent (vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16)
    at vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16
    at vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16
    at fk (vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16)
    at exports.unstable_runWithPriority (vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16)
    at gg (vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16)
    at Oj (vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16)
    at Lj (vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16)
    at vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16
    at exports.unstable_runWithPriority (vendors~main.801c64baa44ce1fed39c.manager.bundle.js:16)

This error seems to be related to the changes made in commit https://github.com/react-theming/storybook-addon/commit/8f9a3caf562ec3284412b30d8ceb3220a382f3b4. 

Could you please suggest if we have to make any changes in the addon usage after the recent upgrade?

**Comment by UsulPro** [comment]
@Vishnupriyaakinapelli we bumped dependencies recently
sorry for the inconvenience but looks like it is worth locking the previous version until we fix it

**Comment by k-harker** [comment]
I was trying to add this plugin to my existing storybook setup when I ran into this issue as well.  Reverting to 1.1.1 has resolved it for me.

In my research I found that the api param was not being passed to setCurrent correctly.  It is being set on `window.api` in the changes linked above so adding a line to the top of `setCurrent` in [actions.js](https://github.com/react-theming/storybook-addon/blob/master/src/actions.js#L4) would patch it, but this doesn't seem like the appropriate fix. 

`if (!api) api = window.api`

Once I put this in place the addon started showing the themes. I've reverted to the previous version for now and I'm looking forward to using this addon. Thanks for your work!
